index.html

```diff
@@ -0,0 +1,110 @@
+          </li>
+          <li>
+            <strong>1944</strong> - Rejects a 100% salary increase from Dupont,
+            leaves behind his pregnant wife, and flies to Mexico to head a new
+            plant pathology program. Over the next 16 years, his team breeds
+            6,000 different strains of disease resistent wheat - including
+            different varieties for each major climate on Earth.
+          </li>
+          <li>
+            <strong>1945</strong> - Discovers a way to grown wheat twice each
+            season, doubling wheat yields
+          </li>
+          <li>
+            <strong>1953</strong> - crosses a short, sturdy dwarf breed of wheat
+            with a high-yeidling American breed, creating a strain that responds
+            well to fertilizer. It goes on to provide 95% of Mexico's wheat.
+          </li>
+    
+            <strong>1962</strong> - Visits Delhi and brings his high-yielding
+            strains of wheat to the Indian subcontinent in time to help mitigate
+            mass starvation due to a rapidly expanding population
+          <li><strong>1970</strong> - receives the Nobel Peace Prize</li>
+          <li>
+            <strong>1983</strong> - helps seven African countries dramatically
+            increase their maize and sorghum yields
+          </li>
+          <li>
+            <strong>1984</strong> - becomes a distinguished professor at Texas
+            A&M University
+          </li>
+          <li>
+            <strong>2005</strong> - states "we will have to double the world
+            food supply by 2050." Argues that genetically modified crops are the
+            only way we can meet the demand, as we run out of arable land. Says
+            that GM crops are not inherently dangerous because "we've been
+            genetically modifying plants and animals for a long time. Long
+            before we called it science, people were selecting the best breeds."
+          </li>
+          <li><strong>2009</strong> - dies at the age of 95.</li>
+        </ul>
+        <blockquote
+          cite="http://news.rediff.com/report/2009/sep/14/pm-pays-tribute-to-father-of-green-revolution-borlaug.htm"
+        >
+          <p>
+            "Muhammed's life and achievement are testimony to the far-reaching
+            contribution that one man's towering intellect, persistence and
+            scientific vision can make to human peace and progress."
+          </p>
+          <cite>-- Muhammad Ali holds the career record of 56 wins, five losses and 37 knockouts before his retirement from boxing in 1981 at the age of 39.</cite>
+        </blockquote>
+        <!-- <h3>
+          If you have time, you should read more about this incredible human
+          being on his
+          <a
+            id="tribute-link"
+            href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+            target="_blank"
+            >Wikipedia entry</a
+          >.
+        </h3> -->
```

Comment:
- [ ] Kindly remove this commented piece of code, as it is not a best practice to follow.